### PR TITLE
Style and include multiple admin boundaries as separate layers

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This Pull Request allows PRISM to read and render multiple admin boundary layers within mapbox. Also, **line** and **fill** mapbox styles are now defined from layers.json configuration file

image

### CODE REVIEW

File: src/config/utils.ts [hunk 1]
```diff
@@ -129,21 +129,29 @@ export const LayerDefinitions: LayersMap = (() => {
...
-export function getBoundaryLayerSingleton(): BoundaryLayerProps {
+export function getBoundaryLayers(): BoundaryLayerProps[] {
   const boundaryLayers = Object.values(LayerDefinitions).filter(
     (layer): layer is BoundaryLayerProps => layer.type === 'boundary',
   );
-  if (boundaryLayers.length > 1) {
+  if (boundaryLayers.length === 0) {
     throw new Error(
-      'More than one Boundary Layer defined! There should only be one boundary layer in layers.json',
+      'No Boundary Layer found! There should be exactly one boundary layer defined in layers.json.',
```
Comment (wadhwamatic) [inline src/config/utils.ts:138]: This is no longer true.

**Comment by wadhwamatic** [comment]
Tested and this works as expected. We should next introduce the concept of levels / order. See https://github.com/WFP-VAM/prism-frontend/issues/292. 

Minor request to update the message around 'only 1 admin boundary file'. See in-line comment https://github.com/WFP-VAM/prism-frontend/pull/306/files#r753441710

File: src/config/utils.ts (hunk 1 above)
Comment (wadhwamatic) [reply src/config/utils.ts:138]: Suggested alternative text:
"No boundary layer found. There should be at least one boundary layer defined in layers.json"

**Review by wadhwamatic — APPROVED** [review]
Good to go. Thanks

**Comment by ericboucher** [comment]
Sorry I am seeing this PR a bit late but it came up while chatting about #338

**Comment by wadhwamatic** [comment]
> Sorry I am seeing this PR a bit late but it came up while chatting about #338
> 
> Can I get a bit more context on why this is needed? What is the excpected behavior for `getBoundaryLayerSingleton`? Should it grab one of the layers? a merge of the layers? something else?

@ericboucher - this was to address https://github.com/WFP-VAM/prism-frontend/issues/284; we need unique styles depending on the admin level

**Pull request merged**